pages/workshop/[wid]/prompt/new.js

```diff
@@ -22,7 +22,7 @@ const NewPrompt = () => {
     }
 
     console.log(data, 'data from create prompt')
```

Comment:
Normally I'd request to remove this console log but going to leave it for now so we can just merge!

Reply:
@MilcahHalili Removed!